src/components/FallenHero.tsx

```diff
@@ -16,10 +16,8 @@ return (
 }
 */
 
-function FallenHero() {
+export function FallenHero({ fallenHero }: { fallenHero: string }) {
   // this component is for rendering individual fallen hero, eventually
   // also with more data, such as scenarios played, image etc.
-  // const {fallenHeroes} = React.useContext(GithubContext);
-
-  return <div className="fallen"> </div>;
+  return <li key={fallenHero}>{fallenHero}</li>;
```

Comment:
Observe the error in the console. Why is it thrown?

Reply:
Because key prop is for controlling and differentiating between component mapped f.e., and not like this. I moved the key   higher

Reply:
The answer is correct, but you did not do it. You only removed the key.

Reply:
goddamn VIktor.. was meant like this, see the change. Must have removed the change by accident.